### ISSUE TITLE
Missing texture on Futura Block

### ISSUE THREAD

**Issue by Napkin** [Chisel-Team/Chisel#250]
Whether I place the block or look at it in JEI, the textures are missing (using Chisel-MC1.9.4-0.0.6.46.jar in MC 1.10.2):
![futura_block](https://cloud.githubusercontent.com/assets/274086/19211496/385bbb5c-8d3e-11e6-8d4e-9bf920a9596c.png)

**Comment by tterrag1098** [comment]
Working for me in .50, can you confirm?

![](http://i.imgur.com/sNAM828.png)

**Comment by Drullkus** [comment]
Issue occurs outside dev env, will investigate personally

**Comment by Napkin** [comment]
As tterrag1098 suggested, I tried Chisel-MC1.9.4-0.0.6.50.jar. Can't get those textures to show up, though.

I tried in a copy of the normal world I play with lots of other mods. I tried in a separate new world with only forge-1.10.2-12.18.1.2095-universal.jar and jei_1.10.2-3.12.5.297.jar. And I tried in a separate new world with the new forge-1.10.2-12.18.2.2099-universal.jar and JEI.

The texture of the Factory Block (reported in issue #236) is fixed but the texture of the Futura Block won't show up.

Thanks for your help!
